### ISSUE TITLE
same issue again with nextjs 13.5.4, with moduleResolution="bundler"

### ISSUE THREAD

**Issue by adarsh-drishya** [cypress-io/cypress#28006]
i am tagging @lmiller1990 please have a look at it.              Not sure, I didn't get a chance to test it thoroughly. Did your commit fix it @jordanpowell88 ?

Does it fix #27448?

Not really ideal, apps with `bundler` will still be bundled slightly differently, but this at least unblocks latest next.js.

_Originally posted by @lmiller1990 in https://github.com/cypress-io/cypress/issues/27484#issuecomment-1672319918_

**Comment by AtofStryker** [comment]
Hi @adarsh-drishya. I am going to close this issue as a duplicate of https://github.com/cypress-io/cypress/issues/27731, but the team is currently working on a fix!
